tests/unit/python/fledge/services/core/api/test_browser_assets.py

```diff
@@ -434,3 +434,52 @@ async def test_bad_asset_bucket_size_and_optional_params(self, client, url, code
         assert 1 == query_patch.call_count
         args, kwargs = query_patch.call_args
         query_patch.assert_called_once_with(payload)
+
+    @pytest.mark.parametrize("request_params, payload", [
+        ('?limit=5&skip=1&order=asc',
+         '{"return": ["reading", {"column": "user_ts", "alias": "timestamp"}],'
+         ' "where": {"column": "asset_code", "condition": "=", "value": "fogbench/humidity"},'
+         ' "skip": 1, "limit": 5, '
+         '"sort": {"column": "user_ts", "direction": "asc"}}',
+         ),
+        ('?limit=5&skip=1&order=desc',
+         '{"return": ["reading", {"column": "user_ts", "alias": "timestamp"}],'
+         ' "where": {"column": "asset_code", "condition": "=", "value": "fogbench/humidity"},'
+         ' "skip": 1,"limit": 5, '
+         '"sort": {"column": "user_ts", "direction": "desc"}}',
+         ),
+        ('?limit=5&skip=1',
+         '{"return": ["reading", {"column": "user_ts", "alias": "timestamp"}],'
+         ' "where": {"column": "asset_code", "condition": "=", "value": "fogbench/humidity"},'
+         ' "skip": 1,"limit": 5, '
+         '"sort": {"column": "user_ts", "direction": "desc"}}',
+         )
+
+    ])
+    async def test_order_payload_good(self, client, request_params, payload):
+        readings_storage_client_mock = MagicMock(ReadingsStorageClientAsync)
+        with patch.object(connect, 'get_readings_async', return_value=readings_storage_client_mock):
+            with patch.object(readings_storage_client_mock, 'query', return_value=mock_coro({'count': 0, 'rows': []})) \
+                    as query_patch:
+                resp = await client.get('fledge/asset/fogbench%2Fhumidity{}'.format(request_params))
+                assert 200 == resp.status
+                r = await resp.text()
+                json_response = json.loads(r)
+                assert [] == json_response
+            args, kwargs = query_patch.call_args
+            assert json.loads(payload) == json.loads(args[0])
+            query_patch.assert_called_once_with(args[0])
+
+    @pytest.mark.parametrize("request_params, response_message", [
+        ('?limit=5&skip=1&order=blah',
+         'order must be asc or desc',
+         ),
+    ])
+    async def test_order_payload_bad(self, client, request_params, response_message):
+        readings_storage_client_mock = MagicMock(ReadingsStorageClientAsync)
+        with patch.object(connect, 'get_readings_async', return_value=readings_storage_client_mock):
+            with patch.object(readings_storage_client_mock, 'query', return_value=mock_coro({'count': 0, 'rows': []})) \
+                    as query_patch:
```

Comment:
Why do you need patch object for this case? As it returns early in the code.
Otherwise we need to ensure in test these never be called. Hence better to remove unused patching.